### PR TITLE
Update tutorials page.

### PR DESCRIPTION
**Pull request by angela24680403**
This PR adds a notebook link to the Intro to AI page, as well as descriptions for the first three tutorial sessions. The icon has been updated, and the naming and numbering of weekly sessions have been fixed.

### CODE REVIEW

**Comment by angela24680403** [comment]
image
This is what the Intro to AI page should look like.

File: our-initiatives/tutorials/index.mdx [hunk 1]
```diff
@@ -25,31 +25,35 @@ Our teaching will be supplemented by engaging AI competitions on [DOXA](https://
...
+
+📚 Access our notebooks and slides here!
 
 <DocCardList />
 
+## Timeline
+
+### Term 1
+
+During the first half term, we aim to cover basic concepts of **classifcal ML**:
```
Comment (TheRootOf3) [inline our-initiatives/tutorials/index.mdx]: Typo, proposed fix below.
```suggestion
During the first half term, we aim to cover basic concepts of **classical ML**:
```

File: our-initiatives/tutorials/regression.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+sidebar_position: 4
+---
+
+# 2: Regression
+
+**Date: 25th October 2023**
+
+💡 **Linear regression** in simple terms is finding the **line of best fit** given a set of data points that loosly follow a linear pattern. One way of doing this is by **minimizes the loss** (difference between actual and computed value), making it useful for **predicting numerical outcomes** based on input features. **Polynomial regression** extends linear regression by allowing the model to capture **nonlinear relationships** between variables, using polynomial functions to fit the data more flexibly. 💡
```
Comment (TheRootOf3) [inline our-initiatives/tutorials/regression.md]: Typo, proposed fix below. Also, I like the explanation that in simple terms it is finding the line of the best fit but this is the case just with 2d feature space. For 3 features that's gonna be a place, for >3 a hyperplane, may be worth specifying that line is just for 2d? Just a note, decision is 100% up to you!
```suggestion
💡 **Linear regression** in simple terms is finding the **line of best fit** given a set of data points that loosely follow a linear pattern. One way of doing this is by **minimising the loss** (difference between actual and computed value), making it useful for **predicting numerical outcomes** based on input features. **Polynomial regression** extends linear regression by allowing the model to capture **nonlinear relationships** between variables, using polynomial functions to fit the data more flexibly. 💡
```

File: our-initiatives/tutorials/visual-computing-1.md [hunk 3]
```diff
@@ -2,4 +2,6 @@
 sidebar_position: 6
```
Comment (TheRootOf3) [inline our-initiatives/tutorials/visual-computing-1.md]: Please change to 7.
```suggestion
sidebar_position: 7
```

File: our-initiatives/tutorials/visual-computing-2.md [hunk 4]
```diff
@@ -2,4 +2,6 @@
 sidebar_position: 7
```
Comment (TheRootOf3) [inline our-initiatives/tutorials/visual-computing-2.md]:
```suggestion
sidebar_position: 8
```

File: our-initiatives/tutorials/rnns.md [hunk 5]
```diff
@@ -2,4 +2,6 @@
 sidebar_position: 8
```
Comment (TheRootOf3) [inline our-initiatives/tutorials/rnns.md]:
```suggestion
sidebar_position: 9
```

File: our-initiatives/tutorials/ml-in-practice.md [hunk 6]
```diff
@@ -2,4 +2,6 @@
 sidebar_position: 9
```
Comment (TheRootOf3) [inline our-initiatives/tutorials/ml-in-practice.md]:
```suggestion
sidebar_position: 10
```

**Review by TheRootOf3 — CHANGES_REQUESTED** [review]
LGTM, 2 typos, some docs ordering and one note, please have a look below!

**Review by TheRootOf3 — APPROVED** [review]
LGTM, well done! 👍

**Pull request merged**